cli/command/stack/cmd.go

```diff
@@ -69,7 +69,7 @@ func NewStackCommand(dockerCli command.Cli) *cobra.Command {
 	flags := cmd.PersistentFlags()
 	flags.String("kubeconfig", "", "Kubernetes config file")
 	flags.SetAnnotation("kubeconfig", "kubernetes", nil)
-	flags.String("orchestrator", "", "Orchestrator to use (swarm|kubernetes|all)")
+	flags.String("orchestrator", "", "Orchestrator to use (swarm|kubernetes)")
```

Comment:
While it's true for commands that have to interact with a specific orchestrator (e.g. deploying, or removing a stack), I think the `--orchestrator=all` option is supported for, `docker stack ls`

@silvin-lubecki (please double check I'm right there)

Reply:
@thaJeztah : So we have this `all` option everywhere other then `stack ls` is of no use ?

Reply:
@thaJeztah : It would be good to know, whether this fix helps or its by design that `all` option is used only with `stack ls` and invalid option in all other stack commands

Reply:
It's by design that it only works for stack ls (for setups that support multiple orchestrators in parallel, which is possible with Docker Enterprise)